src/main/kotlin/step3/Result.kt

```diff
@@ -0,0 +1,13 @@
+package step3
+
+class Result {
+    val records = mutableListOf<Int>()
+
+    fun record(car: Car) {
+        record(car.position)
+    }
+
+    private fun record(position: Int) {
+        records.add(position)
+    }
+}
```

Comment:
Mutable로 정의하고 record를 외부에서 호출해줄 필요가 있을까요?

아래와 같은 형태도 고려해보세요.

```suggestion
class Result(cars: List<Car>) {
    val records = cars.map { it.position }
}
```